src/validation/name.validation.ts

```diff
@@ -0,0 +1,151 @@
+import { NextFunction, Request, Response } from "express";
+import { NameValidationType, ValidationError } from "../model/validation.model";
+import { getField, setBackLink } from "../utils/web";
+import { TITLE_LIST } from "../utils/properties";
+import { DirectorField } from "../model/director.model";
+import { NameValidation } from "./name.validation.config";
+import { createValidationError, createValidationErrorBasic, formatValidationErrors } from "./validation";
+import { Templates } from "../types/template.paths";
+import { urlUtils } from "../utils/url";
+import { Session } from "@companieshouse/node-session-handler";
+import { CURRENT_DIRECTORS_PATH } from "../types/page.urls";
+import { getOfficerFiling } from "../services/officer.filing.service";
+import { lookupWebValidationMessage } from "../utils/api.enumerations";
+import { formerNamesErrorMessageKey } from "../utils/api.enumerations.keys";
+
+const REGEX_FOR_NAMES = /^[-,.:; 0-9A-Z&@$£¥€'"«»''""?!/\\()[\]{}<>*=#%+ÀÁÂÃÄÅĀĂĄÆǼÇĆĈĊČÞĎÐÈÉÊËĒĔĖĘĚĜĞĠĢĤĦÌÍÎÏĨĪĬĮİĴĶĹĻĽĿŁÑŃŅŇŊÒÓÔÕÖØŌŎŐǾŒŔŖŘŚŜŞŠŢŤŦÙÚÛÜŨŪŬŮŰŲŴẀẂẄỲÝŶŸŹŻŽa-zÀÖØſƒǺẀỲàáâãäåāăąæǽçćĉċčþďðèéêëēĕėęěĝģğġĥħìíîïĩīĭįĵķĺļľŀłñńņňŋòóôõöøōŏőǿœŕŗřśŝşšţťŧùúûüũūŭůűųŵẁẃẅỳýŷÿźżž]*$/;
```

Comment:
I'd suggest using quotes (or copying one from API) instead of a RegEx, got my hands burned in occupation validation jira. I have strong reservations that this will not allow spaces and hyphens (-)